recipe/meta.yaml

```diff
@@ -8,31 +8,46 @@ source:
   url: https://github.com/Unidata/netcdf-c/archive/v{{ version }}.tar.gz
   sha256: a2fabf27c72a5ee746e3843e1debbaad37cd035767eaede2045371322211eebb
   patches:
-    - CMakeLists.patch  # [win]
+    # These patches are needed but there is no way I can get them to work on AppVeyor's Windows image :-(
+    - 0001-Fix-bug-Index-exceeds-dimension-bound.patch  # [not win]
+    - 0002-run_get_hdf4_files.sh-Use-curl-if-wget-fails-macOS-d.patch  # [not win]
+    - 0003-Add-find_package-Threads-REQUIRED-to-CMakeLists.txt.patch  # [not win]
+    - 0004-Prefer-getenv-TOPSRCDIR-over-STRINGIFY-TOPSRCDIR.patch  # [not win]
+    - 0005-Convert-some-c99-to-c89-by-hand.patch  # [not win]
+    - 0006-Fix-tests-for-Windows-VS2008-define-__func__-as-__FU.patch  # [not win]
+    - 0007-Fix-broken-test.patch  # [not win]
+    - 0008-Finish-the-missing-code-to-handle-VS-in-test_common..patch  # [not win]
 
 build:
-  number: 3
-  skip: True  # [win and vc<14]
+  number: 4
   run_exports:
     #   https://abi-laboratory.pro/tracker/timeline/netcdf/
     - {{ pin_subpackage('libnetcdf') }}
 
 requirements:
   build:
     - cmake
+    - make  # [not win]
     - pkg-config  # [not win]
+    # Needed for tests: Bash shell not found; disabling shell script tests.
+    - posix  # [win]
+    - m2-m4  # [win]
     - {{ compiler('c') }}
   host:
-    - msinttypes  # [win and vc14]
+    - msinttypes  # [win and vc<14]
+    - bzip2
     - curl
     - zlib
     - hdf4
     - hdf5
+    - jpeg
   run:
+    - bzip2
     - curl
     - zlib
     - hdf4
     - hdf5
+    - jpeg
```

Comment:
I had to add `libssh2  # [win]` here to be able to run `ncdump` etc. Not sure why it is a runtime only requirement....

Reply:
Probably an issue with `curl` that is not pulling that.